codex/erasure/erasure.nim

```diff
@@ -209,12 +205,8 @@ proc decode*(
 
         idxPendingBlocks.del(idxPendingBlocks.find(done))
 
-        without blkOrNone =? (await done), error:
-          trace "Failed retrieving block", exc = error.msg
-          return error.failure
-
-        without blk =? blkOrNone:
-          trace "Block not found"
+        without blk =? await done, error:
```

Comment:
for me, "(await done)" with parentheses makes it more readable

Reply:
Change made.